test/fixtures/minitest_tests.rb

```diff
@@ -1,11 +1,15 @@
 class Test < Minitest::Test
-  def test_method
-  end
+  private def test_private_command; end
```

Comment:
Let's also add the following scenarios (which believe it or not are valid Ruby).
```ruby
private(def test_another_private
end)

public(def test_another_public
end)
```